cmd/config.go

```diff
@@ -75,12 +43,24 @@ func newConfigCommand(config *settings.Config) *cobra.Command {
 		Use:     "validate <path>",
 		Aliases: []string{"check"},
 		Short:   "Check that the config file is well formed.",
-		PreRun: func(cmd *cobra.Command, args []string) {
-			opts.args = args
-			opts.rest = rest.NewFromConfig(GetConfigAPIHost(opts.cfg), config)
-		},
-		RunE: func(cmd *cobra.Command, _ []string) error {
-			return validateConfig(opts, cmd.Flags())
+		RunE: func(cmd *cobra.Command, args []string) error {
+			compiler := config.New(globalConfig)
+			orgID, _ := cmd.Flags().GetString("org-id")
+			orgSlug, _ := cmd.Flags().GetString("org-slug")
+			path := config.DefaultConfigPath
+			if configPath != "" {
+				path = configPath
+			}
+			if len(args) == 1 {
```

Comment:
Are we only ever expecting one arg?

Reply:
As it stands yes based on the help output for the validate and process commands:

```
circleci config validate <path> [flags]
```

Validating this locally outputs: `Error: accepts at most 1 arg(s), received 2`